klayout_package/python/kqcircuits/simulations/simulation.py

```diff
@@ -902,6 +902,8 @@ def create_simulation_layers(self):
             if part.visualise:
                 self.visualise_region(part.region, part.name, f"part_reg_{part.name}")
 
+        # Finally, visualise all ports
+        self.visualise_ports()
     def produce_layers(self, parts):
```

Comment:
```suggestion
        self.visualise_ports()

    def produce_layers(self, parts):
```
Add empty line here please